File: java/control_flow/while_loop/WhileLoopFormatting.java
/*
 * Copyright (C) 2019-2026 Katarzyna Krasińska
 * Languagium - https://github.com/katheroine/languagium
 * Licensed under GPL-3.0 - see LICENSE.md
 */

class WhileLoopFormatting {
  public static void main(String[] args) {
    int i = 0;

    while (i < 10) {
      System.out.println(i + "...");
      i++;
    }

    System.out.println();

    i = 0;

    while (i < 10) {
      System.out.println(i++ + "...");
    }

    System.out.println();

    i = 0;

    while (i < 10)
      System.out.println(i++ + "...");

    System.out.println();

    i = 0;

    while (i < 10) System.out.println(i++ + "...");

    System.out.println();
  }
}

File: pascal/control_flow/while_loop/while_loop_formatting.pas
(*
 * Copyright (C) 2019-2026 Katarzyna Krasińska
 * Languagium - https://github.com/katheroine/languagium
 * Licensed under GPL-3.0 - see LICENSE.md
 *)

program WhileLoopFormatting;

var
  i: integer;

begin
  i := 0;

  while i < 10 do
  begin
    writeln(i, '...');
    i := i + 1;
  end;

  writeln();

  i := 0;

  while (i < 10) do
  begin
    writeln(i, '...');
    i := i + 1;
  end;

  writeln();
end.
